apps/site/lib/site/shuttle_diversion.ex

```diff
@@ -123,6 +124,7 @@ defmodule Site.ShuttleDiversion do
           id: shape.id,
           direction_id: shape.attributes["direction_id"],
           polyline: decode_polyline(shape.attributes["polyline"]),
+          color: hd(shape.relationships["route"]).attributes["color"],
```

Comment:
@digitalcora This builds on the work done in #289 that puts the color onto the route!

Reply:
This would have worked regardless of #289 being merged — that PR added a `color` attribute onto our Elixir struct representation of a `Route`, but this code is operating directly on JSON responses from the API and so doesn't use that.

Reply:
Corrected!